packages/next-app/src/app/stableswapsimulator/analysis/page.tsx

```diff
@@ -43,10 +43,10 @@ export default function Page() {
   }
 
   return (
-    <div className="flex flex-col w-full overflow-auto gap-y-20 pt-8 pr-8 max-h-[calc(100vh-132px)]">
+    <div className="flex max-h-[calc(100vh-132px)] w-full flex-col gap-y-20 overflow-auto pr-8 pt-8">
```

Comment:
I was testing something on my side and it seemed like `max-h-[calc(100vh-132px)]` was making the plots a bit weird on narrower screens (even a bit thinner than the 13" MBP). Have you tried removing it altogether?

Reply:
yup, this PR is focused on the metadata and internal manager side of things. I trying to figure out the plots on stableswap. That's why decided to break into two PR. Sorry for not making that clear!